hbase_sql/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/ExternalTableIT.java

```diff
@@ -2409,15 +2446,15 @@ public void testParquetPartitionColumnName() throws Exception {
     public void testOrcColumnName() throws Exception {
         String tablePath = getExternalResourceDirectory()+"orc_colname";
         methodWatcher.execute(String.format("create external table t_orc (col1 int, col2 varchar(5))" +
-                " STORED AS ORC LOCATION '%s'", tablePath));
+                " STORED AS PARQUET LOCATION '%s'", tablePath));
```

Comment:
Why the change? If this is deliberate we'd need to change the test name too

Reply:
good catch, that wasn't intentional.